src/helpers/dom/element.js

```diff
@@ -762,11 +762,55 @@ export function getComputedStyle(element, rootWindow = window) {
 /**
  * Returns the element's outer width.
  *
+ * @see {preciseOuterWidth}
  * @param {HTMLElement} element An element to get the width from.
  * @returns {number} Element's outer width.
  */
 export function outerWidth(element) {
-  return Math.ceil(element.getBoundingClientRect().width);
+  return element.offsetWidth;
+}
+
+/**
+ * Calculates the current `transform` `scaleX` property value for the passed in
+ * element, relative to the root document. Involves adding a dummy element of a
+ * constant size to the DOM, be cautious about performance.
+ *
+ * @param {HTMLElement} element An element to get the `scaleX` from.
+ * @returns {number} `scaleX` value.
+ */
+export function computeScaleX(element) {
+  // A dummy element with a constant size, used to calculate the `scale`.
+  const $dummy = element.ownerDocument.createElement('div');
```

Comment:
I'd suggest not using the dollar sign. It tells me that the variable contains the jQuery element but it's not. Can we stick to the `dummy` name or `_dummy` if you won't mark the variable as "for special usage"

Reply:
Agree that `$` was often used to mark an element wrapped by jQuery.

In my projects I like to use `dummyEl` to indicate it's an element or node. But recently considering full `dummyElement` and `dummyNode`. Maybe we should discuss a convention for the use across our projects?

Reply:
Renamed it to just `dummy` for now. https://github.com/handsontable/handsontable/pull/7496/commits/623b45269d4aa2cac83ab2472fc7bdd5efa3388e